content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/overview.md

```diff
@@ -53,6 +79,26 @@ Remote procedure calls are how O3DE allows for pushing messages between hosts, r
 
 **Unreliable** RPCs are sent over a "fire and forget" method. The host sending the message has no way to ensure that the message was received.
 
+### Types of RPCs
+
+Similar to network properties, RPCs have two important fields: `InvokeFrom` and `HandleOn`. These fields describe which role the RPC is invoked from and which role it's handled on.
+
+| Field | Description | Values | 
+| - | - | - |
+| `InvokeFrom` | Tells the network which role invokes this RPC. | Server, Autonomous, Client | 
+| `HandleOn` | Tells the network which roles handles this RPC. If Authority, the RPC is handled by the server that has authority over that entity. If Autonomous, the RPC is handled by the player on the local client. If Client, the RPC is handled on all clients. | Authority, Autonomous, Client | 
+
+The following are four types of RPCs with a valid "invoked from and handled on" relationship that you can define, and their possible use cases:
+
+- **Authority-to-Autonomous**: For example, sending corrections about the game state to the user.
+
+- **Authority-to-Client**: Authority sends calls to all the clients. For example, sending information about particle effects. 
+
+- **Server-to-Authority**: This is useful to communicate information between entities. For example, when a player wants to deal damage, Autonomous informs the Server, and the Server sends the DealDamage function to Authority. 
```

Comment:
_This is useful_
Might want to stress this a bit more. _This is required_ for communication between network entities. If you're using a multi-server, entityA (supposing it's owned by ServerA), might be trying to communicate with entityB (supposing it's owned by ServerB). If A talks to B directly, then A will be talking to a proxy, not the real B. Server-to-Authority makes sure that message always finds the entity with authority.